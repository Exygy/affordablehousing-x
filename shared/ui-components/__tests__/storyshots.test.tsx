import initStoryshots, { multiSnapshotWithOptions } from "@storybook/addon-storyshots";
import { mount } from "enzyme";
import { createSerializer } from "enzyme-to-json";

initStoryshots({
  renderer: mount,
  snapshotSerializers: [createSerializer()],
  test: multiSnapshotWithOptions({
    renderer: mount
  })
});
