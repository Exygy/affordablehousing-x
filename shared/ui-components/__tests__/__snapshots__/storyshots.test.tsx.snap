// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button with emoji 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    <span
      aria-label="so cool"
      role="img"
    >
      😀 😎 👍 💯
    </span>
  </button>
</Button>
`;

exports[`Storyshots Button with text 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    Hello Button
  </button>
</Button>
`;

exports[`Storyshots Header header 1`] = `
<PageHeader>
  <header
    className="bg-gray-300 py-10 px-5"
  >
    <h1
      className="title m-auto max-w-5xl"
    />
  </header>
</PageHeader>
`;
